msteams-platform/tabs/how-to/build-adaptive-card-tabs.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Build Adaptive Card Tab
+author: KirtiPereira
+description: Build tabs using Adaptive Card
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+
+# Build Adaptive Card Tab
+
+Use Adaptive Card to build tabs with ease. Build your tab with ready-made UI Lego-blocks that look and feel native on desktop, web, and mobile. Adaptive Card Tab centralizes all Teams app capabilities around a bot backend, thus, eliminating the need for a different backend for your bot and tabs. This will greatly reduce server and maintenance costs of your Teams app.
+This article helps you understand the changes required to be made to the app manifest, how the invoke activity requests and sends information in Adaptive Card Tab, and the impact on the task module workflow. 
+
+## Prerequisites
+
+Before you start using Adaptive Card to build tabs, you must:
+
+* Be familiar with, [bot development](../../bots/what-are-bots.md), [Adaptive Card](../../task-modules-and-cards/what-are-cards.md#adaptive-cards), and [Task Modules](../../task-modules-and-cards/task-modules/task-modules-bots.md) in Teams.
+* Have a bot running in Teams for your development.
+* Have a tenant that is enrolled in ring 1.6.
+
+## Changes to app manifest
+
+Personal apps that render tabs must include a `staticTabs` array in their app manifest. Adaptive Card Tab are rendered when the `contentBotId` property is provided in the `staticTab` definition. Static tab definitions must contain either a `contentBotId`, specifying an Adaptive Card Tab or a `contentUrl`, specifying a typical hosted web content tab experience.
+
+> [!NOTE]
+> The `contentBotId` property is currently available in the developer preview manifest schema only. 
+
+Provide the `contentBotId` property with the `botId` the Adaptive Card Tab must communicate with. The `entityId` configured for the Adaptive Card Tab is sent in the `tabContext` parameter of each invoke request, and can be used to differentiate different Adaptive Card Tab that are powered by the same bot. For more information about other static tab definition fields, see [manifest schema](../../resources/schema/manifest-schema.md#statictabs).
+
+Following is a sample Adaptive Card Tab manifest:
+
+```json
+{
+  "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "id": "4b0b7c70-d2bc-11ea-a813-bfc190db5187",
+  "version": "0.0.1",
+  "packageName": "acprototype",
+  "developer": {
+    "name": "proto",
+    "websiteUrl": "https://acprototype.azurewebsites.net",
+    "privacyUrl": "https://acprototype.azurewebsites.net/privacy.html",
+    "termsOfUseUrl": "https://acprototype.azurewebsites.net/tou.html"
+  },
+  "name": {
+    "short": "Workday",
```

Comment:
We should probably remove references to Workday in this sample manifest

Reply:
@lokimeyburg - to share the update.